script/zonemaster_backend_testagent

```diff
@@ -20,6 +20,8 @@ BEGIN {
 	$ENV{PERL_JSON_BACKEND} = 'JSON::PP';
 }
 
+$|++;
```

Comment:
Have you considered writing this like `use English; $OUTPUT_AUTOFLUSH = 1;` and/or adding an explanatory comment? That's far more helpful to both the non-native Perl speaker and search engines.